views/projects/create.erb

```diff
@@ -0,0 +1,55 @@
+<div class="container">
+  <form action='/projects' name="createProject" method='post' onsubmit="return validateForm()">
+    <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>New Name</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Project name</td>
+        <td>
+           <input type="text" name="name" placeholder="New project">
+        </td>
+      </tr>
+      <tr>
+        <td>Client Id</td>
+        <td>
+          <select  name="client_id">
+          <% for @client in @clients %>
+             <% if @client.active %>
+                <option value="<%= @client.id %>"><%= @client.id %></option>
```

Comment:
Lets show the user the `@client.name` than ID.
So here I would change the second `@client.id` to `@client.name`